react/week2/todo-list/src/Counter.js

```diff
@@ -0,0 +1,35 @@
+import React, {Component} from 'react';
+
+class Counter extends Component {
+  constructor(){
+    super()
+    this.state = {counter: 0}
+  };
+  
+  componentDidMount = () => {
+    this.timer = setInterval(
+      () => this.incrementCounter(),
+    1000
+    );
```

Comment:
If you think about it, `incrementCounter` is already an arrow function, so you could pass this in simple as a reference to the function like: 
```suggestion
    this.timer = setInterval(this.incrementCounter, 1000);
```